### PR TITLE
Ddcnos 5425

### PR DESCRIPTION
**Pull request by andydow311**
changes so that particular 403 and 400 ReportLifeEventErrors are not reported as 500s.

### CODE REVIEW

File: app/uk/gov/hmrc/lisaapi/connectors/DesConnector.scala [hunk 1]
```diff
@@ -162,7 +162,7 @@ class DesConnector @Inject()(
   }
 
   /**
-    * Attempts to close a LISA account
+    * Attempts to report a LISA account
```
Comment (pascal22p) [inline app/uk/gov/hmrc/lisaapi/connectors/DesConnector.scala]: Change from close to report?
Please double check the method is closeAccount

Comment (andydow311) [reply app/uk/gov/hmrc/lisaapi/connectors/DesConnector.scala]: Don't recall making this change - must have slipped in. I'll revert it back to close.

**Comment by danewatts** [comment]
Whoever merges this PR please use Squash and Merge, thanks

**Comment by vajarova** [comment]
Please don't merge until passed TESTING

**Pull request merged**